### PR TITLE
Switch to Github Actions

### PR DESCRIPTION
**Pull request by jderusse**
IIRC you once told me that you want to switch to GA. Here is a config I use in https://github.com/jderusse/http-broadcast.

Feel free to pick/update

### CODE REVIEW

File: .github/workflows/coverage.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  test:
+    name: Test with Coverage
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@master
+    - name: Set up Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.12
```
[the remark this answers is not in the document]
Comment (jderusse) [reply .github/workflows/coverage.yaml]: not yet. Master branch is running 1.12 (see go.mod)

**Comment by jderusse** [comment]
Do not merge YET. I've to adapt build script if you merge #197

**Comment by jderusse** [comment]
PR updated to use golangci-lint.
I don't know why my fork did not triggered the build.

**Comment by dunglas** [comment]
Merged in #260.

**Pull request closed without merging**